internal/rootcoord/task.go

```diff
@@ -108,6 +117,11 @@ func (t *CreateCollectionReqTask) Execute(ctx context.Context) error {
 		zap.Int32("ShardsNum", t.Req.ShardsNum),
 		zap.String("ConsistencyLevel", t.Req.ConsistencyLevel.String()))
 
+	if hasSystemFields(&schema, []string{RowIDFieldName, TimeStampFieldName}) {
```

Comment:
better to change the definition of RowIDFieldName and TimeStampFieldName to uncommon strings such as __system_rowid and __system_ts. Currently, they are "RowID" and "Timestamp"

@xiaofan-luan

Reply:
__sys_rowid
__sys_ts